pybossa/otp.py

```diff
@@ -73,3 +73,10 @@ def retrieve_email_for_token(token):
 def expire_token(token):
     key = _create_url_token_key(token)
     conn.delete(key)
+
+def is_enabled(user_email, config):
+    # Allow bypassing two-factor-auth if the feature is disabled or if the user's email is included in the bypass list.
+    bypass_list = getattr(config, 'BYPASS_TWO_FACTOR_AUTH', [])
```

Comment:
You should be able to do `config.get('BYPASS_TWO_FACTOR_AUTH', [])`

Reply:
Thank you. Yes, I realized this well from the web unit test failing. Working on an update. :)